packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -52,7 +52,7 @@ class EditableString implements KeyboardClient {
   }
 
   String textAfter(TextRange range) {
-    return text.substring(range.end);
+    return text.substring(range.end, text.length);
```

Comment:
End is exclusive, but you're right this is the default behavior. So the change is harmless but irrelevant.